<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configdlg</class>
 <widget class="QDialog" name="configdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>441</width>
     <height>411</height>
    </rect>
   </property>
   <attribute name="headerDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>441</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>Manual  Setting</string>
   </property>
   <widget class="QPushButton" name="btnGetdata">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Get data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leValue">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetdata">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px;</string>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
